.github/workflows/daily-boot-iso-rhel8.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and test daily RHEL boot.iso
+on:
+  schedule:
+    - cron: 0 3 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: run
+    runs-on: [self-hosted, kstest]
+    steps:
+      # self-hosted runners don't do this automatically; also useful to keep stuff around for debugging
+      # need to run sudo as the launch script and the container create root/other user owned files
+      - name: Clean up previous run
+        run: |
+          sudo podman ps -q --all --filter='ancestor=kstest-runner' | xargs -tr sudo podman rm -f
+          sudo podman volume rm --all || true
+          sudo rm -rf *
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Build boot.iso with lorax
+        run: |
+          mkdir build
+          # HACK: lorax and podman overlayfs really hate each other: https://bugzilla.redhat.com/show_bug.cgi?id=1906364
+          sudo podman run --rm -i --storage-driver=vfs --root /tmp/podman-vfs --privileged --device /dev/loop0 --device /dev/loop-control --network host -v $PWD/build:/data registry.access.redhat.com/ubi8 <<EOF
+          set -eux
+
+          # lorax must run on RHEL host to produce RHEL guest images :-(
+          # need to enable internal repos so that we don't need a subscription;
+          rm /etc/yum.repos.d/*
+          cat <<EOR > /etc/yum.repos.d/rhel8.repo
+          [RHEL-8-BaseOS]
+          name=baseos
+          baseurl=http://download.devel.redhat.com/nightly/rhel-8/RHEL-8/latest-RHEL-8/compose/BaseOS/x86_64/os/
+          enabled=1
+          gpgcheck=0
+          install_weak_deps=0
+          module_hotfixes=1
+
+          [RHEL-8-AppStream]
+          name=appstream
+          baseurl=http://download.devel.redhat.com/nightly/rhel-8/RHEL-8/latest-RHEL-8/compose/AppStream/x86_64/os/
+          enabled=1
+          gpgcheck=0
+          install_weak_deps=0
+          module_hotfixes=1
+          EOR
+
+          dnf install -y lorax
+
+          cd /data
+          lorax -p "Red Hat Enterprise Linux" \
+             --version 8 --release 8 --variant=BaseOS --volid=RHEL-8-4-0-BaseOS-x86_64 --nomacboot \
```

Comment:
I should validate whether we can change the volume ID to something like "8-daily". I don't like the "8-4" hardcoding here. It should work, I just renamed it to that in case that was the reason for the broken boot.

Reply:
Ack, this [does work](https://github.com/rhinstaller/kickstart-tests/runs/1532875634?check_suite_focus=true) (some network flakes, but if the labeling was broken, all the tests would fail).